contrib/crane/template.py

```diff
@@ -0,0 +1,43 @@
+pkgname = "crane"
+pkgver = "0.20.1"
+pkgrel = 0
+build_style = "go"
+make_build_args = [
+    "./cmd/crane",
+]
+hostmakedepends = [
+    "go",
+]
+pkgdesc = "Container image and registry manipulation tool"
+maintainer = "Radosław Piliszek <radek@piliszek.it>"
+license = "Apache-2.0"
+url = "https://github.com/google/go-containerregistry"
+source = f"{url}/archive/refs/tags/v{pkgver}.tar.gz"
+sha256 = "4bae53f34011e35ef874a60123b8ae70a5e992d804decb030479dbb888afe6d1"
+
+
+_supported_shells = ["bash", "zsh", "fish"]
+
+
+# NOTE(yoctozepto): docs are present but they do not render properly
+# as manpages after using go-md2man, so skipping them. Built-in
+# command help has the same content as is typical in cobra-based go programs.
+def post_build(self):
+    for shell in _supported_shells:
+        self.do(
```

Comment:
you don't need sh, just `.do()` the `crane` binary

Reply:
e.g.

```
for shell in ["bash", "zsh", "fish"]:
    with open(self.cwd / f"crane.{shell}", "w") as cf:
        self.do(
            "./crane",
            "completion",
            shell,
            stdout=cf,
        )

Reply:
will do